src/game/game.cpp

```diff
@@ -0,0 +1,40 @@
+#include "game.hpp"
+
+using namespace std;
+using namespace chrono;
+
+Game::Game(int64_t updatesPerSecond)
+{
+  this->updatesPerSec = updatesPerSecond;
+}
+
+Game::~Game()
```

Comment:
This can be left out as the compiler will generate a default destructor. A good principle is to follow the rule of five: https://cpppatterns.com/patterns/rule-of-five.html.

Reply:
:ok_hand: